src/renderer/components/download-settings/download-settings.sass

```diff
@@ -0,0 +1,8 @@
+@use "../../sass-partials/settings"
+
+@media only screen and (max-width: 500px)
+  .downloadSettingsFlexBox
+    justify-content: flex-start
+
+.folderDisplay
+  width: 50vh
```

Comment:
I know this is a probably minor issue but this project does have setting to ensure final new line

https://github.com/FreeTubeApp/FreeTube/blob/development/.editorconfig#L9